### DISCUSSION TITLE
\cite时出现no bib files found 的错误

### DISCUSSION THREAD

**Discussion by Chang-007** [note286/xduts#166] [Q&A] [answered]
使用texlive+sublime，在\cite自动填充时弹出no bib files found的错误，路径没错，bib文件也有内容，很慌张
![PixPin_2024-03-08_12-46-20](https://github.com/note286/xduts/assets/55450113/e9170e78-2029-4b9f-83d1-91c48d5dfcc3)
![PixPin_2024-03-08_12-46-48](https://github.com/note286/xduts/assets/55450113/211b72b2-0338-4b91-a89b-cc7f0394f62e)

**Comment by note286** [comment, marked as answer]
参考 #157

**Reply by Chang-007** [reply]
感谢解答！已解决
